lamindb/_annotate.py

```diff
@@ -920,10 +935,11 @@ def validate_categories_in_df(
 
 def save_artifact(
     data: pd.DataFrame | ad.AnnData | MuData,
-    description: str,
     fields: dict[str, FieldAttr] | dict[str, dict[str, FieldAttr]],
     columns_field: FieldAttr | dict[str, FieldAttr],
+    description: str | None = None,
     organism: str | None = None,
+    adata: ad.AnnData | None = None,
```

Comment:
Why can we pass `adata` and `data` here? I think both should be removed for clarity.

Reply:
This is not user facing, and I need to pass the path and the backed object in order to register artifact and feature sets